### DISCUSSION TITLE
How to trace the syscalls in docker container?

### DISCUSSION THREAD

**Discussion by wangpei-nice** [bpftrace/bpftrace#1988] [Development]
Hi bpftrace maintainers
I find the following example in bpftrace README,  but there is no anything about container like mycg in /sys/fs/cgroup/unified/.

```
#Files opened, for processes in the root cgroup-v2
bpftrace -e 'tracepoint:syscalls:sys_enter_openat /cgroup == cgroupid("/sys/fs/cgroup/unified/mycg")/ { printf("%s\n", str(args->filename)); }'
```

![image](https://user-images.githubusercontent.com/81410079/130888196-76af49b5-dae4-4e0f-bd27-2d25dd51fabf.png)
![image](https://user-images.githubusercontent.com/81410079/130888294-bf865424-c9b9-4af3-90c6-b9d751d6dfdd.png)

**Comment by fbs** [comment]
`mycg` is short for `my cgroup` i.e the cgroup you want to trace. For this you need cgroupv2. Not sure if docker does that by default so you might have to enable it. Docker cgroups are named after the container id afaik so you  can try `find /sys/fs/cgroup -name '$CONTAINERID'`

**Reply by wangpei-nice** [reply]
Thanks for your answer!
